lib/Reply.js

```diff
@@ -192,6 +186,16 @@ function cannonicalizeDirective(directive) {
   // Legacy support.
   // We used to support this slightly altered from spec version of the play directive.
   // We'll check if we think that's still happening and transform to the full Alexa directive
+  if (_.isString(directive.hint) && !_.isEmpty(directive.hint)) {
```

Comment:
@rmberrios i think this comments are wrong

Reply:
@armonge  That's related to the play directive. I think we should take it off